app/assets/javascripts/app/merge_order_form.js.coffee

```diff
@@ -0,0 +1,16 @@
+class window.MergeOrderForm
+  constructor: (@$form) ->
+    @$fulfilled_at = @$form.find("#fulfilled_at")
+    @$order_status_id = @$form.find("#order_status_id")
+    @initializeListener()
+    @$order_status_id.change()
+
+  initializeListener: ->
+    @$order_status_id.change (event) =>
+      if $(event.target).find("option:selected").text() == "Complete"
```

Comment:
It does, but `val()` gives the `order_status_id` value, not the text that I want here.